MC/config/PWGHF/pythia8/generator/pythia8_charmhadronic_XiC_NoDecay.cfg

```diff
@@ -38,5 +38,11 @@ BeamRemnants:saturation 5
 #OmegaC
 4332:tau0 = 0.0803
 
-### switch off XiC decays
+### switch off XiC0 decays
 4132:onMode = off
+
```

Comment:
I would define a separated config for Xic0, Xic+, and OmegaC0 (one for each `ini`)